enclave/asym_keys.c

```diff
@@ -551,3 +553,196 @@ void oe_free_key(
         oe_free(key_info);
     }
 }
+
+static oe_result_t _copy_to_from_host(
+    bool to_host,
+    const void* data,
+    size_t data_size,
+    uint8_t** out)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    uint8_t* out_local = NULL;
+
+    if (data == NULL)
+    {
+        *out = NULL;
+        OE_RAISE(OE_OK);
```

Comment:
OE_RAISE(success) is discouraged. It will emit unnecessary logs.

Reply:
Fixed.